src/base/error_code.h

```diff
@@ -1630,7 +1630,7 @@
 #define ER_NO_PAGE_SERVER_CONNECTION                -1269
 #define ER_TOOL_INVALID_WITH_REMOTE_STORAGE         -1270
 #define ER_COMMAND_INVALID_WITH_REMOTE_STORAGE      -1271
-#define ER_SCALABILITY_DEV_RESERVED_ERROR09         -1272
+#define ER_NET_TRANSACTIONSERVER_CONNECTION         -1272
```

Comment:
`TRANSACTIONSERVER` is too long and unreadable.
```suggestion
#define ER_NET_TRANSACTION_SERVER_CONNECTION         -1272
```

Reply:
added the missing _